OpenEdXMobile/src/main/java/org/edx/mobile/base/BaseAppActivity.java

```diff
@@ -49,7 +51,8 @@ public boolean onCreateOptionsMenu(Menu menu) {
                     googleCastDelegate.showIntroductoryOverlay(this, mediaRouteMenuItem);
                 }
             }
-        } catch (Exception ignore) {
+        } catch (Exception e) {
+            logger.error(e);
```

Comment:
This won't log the exception to Crashlytics as a non-fatal, you'll have to call the function as follows:
` logger.error(e, true);`